charts/datadog/templates/system-probe-configmap.yaml

```diff
@@ -210,14 +210,19 @@ data:
             "stat64",
             "statfs",
             "sysinfo",
+            "tgkill",
             "umask",
             "uname",
             "unlink",
             "unlinkat",
             "wait4",
             "waitid",
             "waitpid",
-            "write"
+            "write",
+            # The following system calls are only necessary for kubectl exec
```

Comment:
I think this comment is screwing things up, because it is inside a JSON config file, and JSON doesn't allow comments.